src/pybind/Makefile

```diff
@@ -97,6 +97,12 @@ LDFLAGS += -Wl,-rpath=$(CURDIR)/../../tools/openfst/lib
 
 LIBFILE=$(LIBNAME)$(LIBFILE_EXTENSION)
 
+ifdef MKL_LD_PRELAOD
+EMPTY :=
+SPACE := $(EMPTY) $(EMPTY)
```

Comment:
Does `SPACE := " "` do the same?

Reply:
No.

You can use `$(info $(some_variable))` to view the value of `some_variable` in `Makefile`.

Neither can you write
```
SPACE := $(EMPTY)
```

**nor**
```
SPACE := $(EMPTY) $(EMPTY) $(EMPTY)
```

It has to be exactly two `$(EMPTY)`. I do NOT know why but it works!

Reply:
Hmm... I tried `SPACE := " "` and tests passed..

Reply:
Did you set the LD_PRELOAD environment variable?

Reply:
No

Reply:
`echo $LD_PRELOAD` is empty

Reply:
I just tried `SPACE := " "` and the tests also passed.

But if you print the value of `LD_PRELOAD`, it is not colon `:` separated. I do
not know why spaces sperated values also work for `LD_PRLOAD`.

What I mentioned above is that you can only get colon separated `LD_PRELOAD`
using the aforementioned approached.

Reply:
If you use `SPACE := " "`, then please put a statement right after `LD_PRELOAD +=`
```
$(error $(LD_PRELOAD))
```
you will see that it prints a list of space separated paths.

Reply:
OK. np

Reply:
mark as resolved.